README.md

```diff
@@ -126,6 +128,7 @@ A curated list of tools and resources for Platform Engineering.
 - [Platform Engineering story from a CTO: WHY, WHAT, HOW](https://medium.com/agorapulse-stories/platform-engineering-part-1-why-the-evolution-of-developer-cognitive-load-9f36f5cc2888)
 - [Create Preview Environments with Terraform, GitHub Actions, and Vercel](https://developer.hashicorp.com/terraform/tutorials/applications/preview-environments-vercel)
 - [Guide To Internal Developer Portals](https://www.getport.io/blog/guide-to-internal-developer-portals)
+- [The Internal Developer Platform Revolution: what every CTO should know](https://blog.mia-platform.eu/en/the-internal-developer-platform-revolution)
```

Comment:
As was mentioned in https://github.com/shospodarets/awesome-platform-engineering/pull/3/files#r1116642877
Can you please remove, since main content is covered by other intro articles in this section, and this one more pointing to MIA platform, which is added in another part of this PR